src/components/Watch.svelte

```diff
@@ -98,23 +104,22 @@
   height: calc(100% - 40px);"
 >
   <MaterialApp theme="dark">
-    {#if $displayMode === DisplayMode.FULLPAGE && $showCaption}
+    {#if isFullPage && $showCaption}
       <Captions />
     {/if}
     <div
       id="mainUI"
       class="flex 
-        {$videoSide == VideoSide.TOP ? 'horizontal' : 'vertical'} 
-        {$videoSide == VideoSide.RIGHT ? 'reversed' : ''}"
+        {isTopSide ? 'horizontal' : 'vertical'} 
+        {isRightSide ? 'reversed' : ''}"
```

Comment:
Please use ``class:`` with this. If you don't want to set two class directives, you can just set a default css value and set a class that overrides the default one.